app/assets/javascripts/components/standards/standards_row.js.coffee

```diff
@@ -0,0 +1,65 @@
+{div, tr, td, button} = React.DOM
+
+window.StandardsRowClass = React.createClass
+
+  displayName: "StandardsRowClass"
+
+  handleButton: (e) -> 
+
+    console.log("INFO adding standard", @props.statement.uri, @props.material.material_type, @props.material.material_id);
+
+    statement   = @props.statement
+    material    = @props.material
+    apiUrl      = null
+
+    params      = { identifier:     statement.uri,          \
+                    material_type:  material.material_type, \
+                    material_id:    material.material_id }
+
+    if !statement.is_applied
+      apiUrl = "/api/v1/materials/add_materials_standard"
+    else
+      apiUrl = "/api/v1/materials/remove_materials_standard"
+
+    jQuery.ajax
+      url: apiUrl
+      data: params
+      dataType: 'json'
+      success: (data) =>
```

Comment:
This should specify the post verb. This is further evidence that get requests are working for routes specified as `post`.  If you have time it would be worth figuring out why. The reason for using post is so that pages on other domains cannot send requests to this route (unless we enable CORS support for this route.)